docs/content/en/docs/operator-manual/piped/adding-a-cloud-provider.md

```diff
@@ -103,3 +103,30 @@ It attempts to retrieve credentials in the following order:
 Therefore, you don't have to set credentialsFile if you use the environment variables or the EC2 Instance Role. Keep in mind the IAM role/user that you use with your Piped must possess the IAM policy permission for at least `Lambda.Function` and `Lambda.Alias` resources controll (list/read/write).
 
 See [ConfigurationReference](/docs/operator-manual/piped/configuration-reference/#cloudproviderlambdaconfig) for the full configuration.
+
+### Configuring ECS cloud provider
+
+Adding a ECS provider requires the region name where ECS cluster is running.
+
+```yaml
+apiVersion: pipecd.dev/v1beta1
+kind: Piped
+spec:
+  ...
+  cloudProviders:
+    - name: ecs-dev
+      type: ECS
+      config:
+        region: {ECS_CLUSTER_REGION}
+        profile: default
+        credentialsFile: {PATH_TO_THE_CREDENTIAL_FILE}
+```
+
+Just same as Lambda applications' credentials methods, there are several ways to authorize Piped agent to enable it performs deployment jobs.
```

Comment:
nit:  How about this
`Just same as Lambda cloud provider, `

Reply:
Addressed on https://github.com/pipe-cd/pipe/pull/2080/commits/b5f3e68d674fc950bd733abcf071688f01034a07 🙏